rainforest-cli.gemspec

```diff
@@ -18,10 +18,10 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "httparty"
-  spec.add_dependency "parallel"
-  spec.add_dependency "ruby-progressbar"
-  spec.add_dependency "rainforest"
+  spec.add_dependency "httparty", "0.13.7"
```

Comment:
I think all of these should be `~>`. Otherwise they might mess up dependency fetching for projects that require it.